README.md

```diff
@@ -0,0 +1,73 @@
+# Flatpak packaging for Delta Chat Desktop
+
+## Building
+
+If you'd like to locally build this flapak, you'll need both `flatpak`
+and `flatpak-builder` installed.  E.g. on Debian you can run `apt
+install flatpak flatpak-builder` to install these tools.
+
+### flatpak dependencies
+
+You need to install the required flatpak runtime, SDK and electron
+BaseApp.  At the time of this writing they are the 18.08 versions, but
+check the `chat.delta.desktop.yml`'s `runtime-version` and
+`base-version` properties for the current version numbers.
+
+```
+flatpak install flathub \
```

Comment:
given the level of detail, it might be worth linking towards https://flatpak.org/setup/ and maybe even copy and pasting the line to add the "flathub" remote:
`  flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo`

Reply:
Good idea, will do